src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -1,17 +1,27 @@
 (function (global, doc, eZ, $) {
+    let lastInsertTooltipTarget = null;
     const TOOLTIPS_SELECTOR = '[title]';
     const observerConfig = {
         childList: true,
         subtree: true,
     };
     const observer = new MutationObserver((mutationsList) => {
-        mutationsList.forEach((mutation) => {
-            const showedTooltipNode = doc.querySelector('.ez-tooltip.show');
+        if (lastInsertTooltipTarget) {
+            mutationsList.forEach((mutation) => {
+                const { removedNodes } = mutation;
 
-            if (mutation.removedNodes.length && showedTooltipNode) {
-                showedTooltipNode.remove();
-            }
-        });
+                if (removedNodes.length) {
+                    removedNodes.forEach((removedNode) => {
+                        if (removedNode.classList && !removedNode.classList.contains('ez-tooltip')) {
```

Comment:
is this `removedNode.classList` needed? classList is DOM property so it should always exist